jmix-gridexport/gridexport-flowui/src/main/java/io/jmix/gridexportflowui/exporter/entitiesloader/KeysetAllEntitiesLoader.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.gridexportflowui.exporter.entitiesloader;
+
+import io.jmix.core.*;
+import io.jmix.core.metamodel.model.MetaClass;
+import io.jmix.core.querycondition.Condition;
+import io.jmix.core.querycondition.LogicalCondition;
+import io.jmix.core.querycondition.PropertyCondition;
+import io.jmix.flowui.model.CollectionLoader;
+import io.jmix.gridexportflowui.GridExportProperties;
+import io.jmix.gridexportflowui.exporter.EntityExportContext;
+import io.jmix.gridexportflowui.exporter.EntityExporter;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.PlatformTransactionManager;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * This loader implements keyset pagination strategy.
```

Comment:
Let's add a couple sentences that describe each pagination strategy. You may take some definitions ideas from here: https://dev.to/pragativerma18/unlocking-the-power-of-api-pagination-best-practices-and-strategies-4b49